File: JWP_API/JadeWesserPort/DTOs/VVNDTOs/LoadVVNCreateDTO.cs
using JadeWesserPort.DTOs.CargoManifestDTOs;

namespace JadeWesserPort.DTOs.VVNDTOs;

public class LoadVVNCreateDTO : VVNCreateDTO
{
    public LoadCargoManifestDTO CargoManifestDTO { get; set; } = null!;
}

File: Frontend/src/infrastructure/dtos/vvn/loadVVNCreateDto.ts
import type { LoadCargoManifestDto } from "../cargo-manifest/loadCargoManifestDto";
import type { VVNCreateDto } from "./vvnCreateDto";

export type LoadVVNCreateDto = VVNCreateDto & {
  cargoManifestDTO: LoadCargoManifestDto;
};
